aspnetcore/fundamentals/logging/index/samples/3.x/CustomLogger/ColoredConsoleLogger/ColoredConsoleLogger.cs

```diff
@@ -22,7 +22,7 @@ public IDisposable BeginScope<TState>(TState state)
 
         public bool IsEnabled(LogLevel logLevel)
         {
-            return logLevel == _config.LogLevel;
+            return logLevel >= _config.LogLevel;
```

Comment:
As mentioned in a comment of the review by @BrennanConroy this would enable the the output for a log level higher or equal to the configured one, instead of a specific one. This should be reflected in the text.

Reply:
I call that out in the text.